v3/pkg/application/popupmenu_windows.go

```diff
@@ -131,6 +132,12 @@ func (p *Win32Menu) buildMenu(parentMenu w32.HMENU, inputMenu *Menu) {
 				}
 			}
 		}
+
+		// If the item is hidden, don't append
+		if item.Hidden() {
+			continue
```

Comment:
Do we still need to set the menu icon in this case?

Reply:
No I don't think so